keyboards/nopunin10did/railroad/railroad.h

```diff
@@ -0,0 +1,44 @@
+#include "quantum.h"
+
+// The Railroad Keyboard
+// Copyright 2020 by W. Alex Ronke, a.k.a. NoPunIn10Did (w.alex.ronke@gmail.com)
+// Licensed under GNU GPL v2, as per the QMK firmware project
+
+#ifndef RAILROAD_H
+#define RAILROAD_H
+
```

Comment:
```suggestion
/* Copyright 2020 W. Alex Ronke, a.k.a. NoPunIn10Did (w.alex.ronke@gmail.com)
 *
 * This program is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 2 of the License, or
 * (at your option) any later version.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */

#pragma once

#include "quantum.h"
```

Fuller license declaration, `#pragma once` instead of older-style include guards and at the top of the page, `#include "quantum.h" moved below `#pragma once`.